src/backends/perf_pt/mod.rs

```diff
@@ -416,7 +427,9 @@ impl PerfPTTracer {
         if !power_of_2(config.aux_bufsize) {
             return Err(HWTracerError::BadConfig(String::from("aux_bufsize must be a positive power of 2")));
         }
-
+        if config.aux_copyout_threshold < 0. || config.aux_copyout_threshold > 1. {
```

Comment:
Shouldn't this error be in the `aux_copyout_threshold` function?

Reply:
That's a copy-pasta, indeed.